content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,249 @@
+  must be a PDF. To remove the document, click the clear icon (trash can icon).
+
+Once the edits are complete, click the :guilabel:`Save` button. All the information for the selected
+contract template will populate the fields in the :guilabel:`Salary Information` tab. Any additional
+tabs, such as :guilabel:`Personal Documents`, will appear if applicable.
+
+Salary Information
+------------------
+
+.. image:: contracts/salary-info.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+This section is where the specific salary details are defined. This section is country-specific, so
+depending on where the company is located, these fields will vary.
+
+Enter the amount in the various fields, or check a box to apply a benefit. Some options that can be
+entered here include :guilabel:`Meal Vouchers`, :guilabel:`Fuel Card`, :guilabel:`Internet`,
+:guilabel:`Paid Time Off`, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the
+:guilabel:`Contract Details` tab.
+
+Attachment of Salary
+--------------------
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click on :guilabel:`Add a line`. Type in a description for the allocation
+under :guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop-down menu. Chose from:
+
+- :guilabel:`Assignment of Salary` - Any payments taken out towards something that is *not* child
+  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
+- :guilabel:`Allocaiton of Salary` - Any deduction that is not required but voluntary, such as a
+  pre-tax allocation to a college savings account.
+- :guilabel:`Child Support` - Any payments taken out specifically for child support.
+
+Enter the start and end dates the entry applies to by clicking on the drop-down menu under
+:guilabel:`From` and :guilabel:`To`, navigating to the correct month and year by using the arrow
+icons, then clicking on the date.
+
+Last, enter the :guilabel:`amount` that each payslip will pay towards the entry.
+
+Delete a line by clicking on the trash can icon at the end of the line.
+
+.. image:: contracts/delete-garnishment.png
+   :align: center
+   :alt: Remove a new line by clicking the delete icon.
+
+Save and send the contract
+--------------------------
+
+Once contracts have been created and/or modified, save the contract by clicking the :guilabel:`Save`
+smart button. Next, the contract must be sent to the employee to be signed.
+
+Click on one of the following smart buttons:
```

Comment:
These are regular buttons as well. Also adding a little bit of context to this sentence.
```suggestion
Click on one of the following buttons to send the contract to the employee:
```